util/latlng_recon_service.py

```diff
@@ -16,13 +16,34 @@
 See latlng_recon_service_test.py for usage example.
 """
 
+import concurrent.futures
```

Comment:
Let's pull out the changes to this library to a new PR. The PR can stand alone without it, and I'd prefer more eyes on this change. Some of our users are on slower connections so we'd want the changes to work for them too.

Reply:
Moved to https://github.com/datacommonsorg/data/pull/779